packages/wonder-blocks-tooltip/components/tooltip-content.js

```diff
@@ -0,0 +1,49 @@
+// @flow
+import * as React from "react";
+import {View} from "@khanacademy/wonder-blocks-core";
+import {HeadingSmall, LabelMedium} from "@khanacademy/wonder-blocks-typography";
+import type {Typography} from "@khanacademy/wonder-blocks-typography";
+
+type Props = {|
+    title?: string | React.Element<Typography>,
```

Comment:
Question: Do we want to allow any old `Text` as well? Currently we're limiting to `Typography` components only. Just wondering if we should have the escape hatch of allowing any `Text` for these props.